Autopilot/boardfiles/Src/tim.c

```diff
@@ -95,6 +96,44 @@ void MX_TIM11_Init(void)
 
 }
 
+/* TIM12 Init Function */
+void MX_TIM12_Init(void)
```

Comment:
Ignore this if you have already configured this timer using CubeMX

Right now you have created these timers, but I don't think our chip will be able to run them since we haven't configured them. What you will have to do is use CubeMX (https://www.st.com/en/development-tools/stm32cubemx.html) and add your configurations to our `Autopilot_ZeropilotRev1.ioc` file found in `./boardfiles`. 

After configuration, thse methods will be regenerated and you will need to verify that they are correct. You will also need to include the `.ioc` file in this PR. 

![Screen Shot 2021-06-18 at 6 21 44 PM](https://user-images.githubusercontent.com/52176616/122620872-0ee20d00-d062-11eb-828b-b4c16dc03b0d.png)

Reply:
Sounds good - I do remember having trouble with the .ioc file in the past but I'll let you know asap if I run into troubles